generators/server/index.js

```diff
@@ -63,7 +63,7 @@ module.exports = class JHipsterServerGenerator extends BaseBlueprintGenerator {
           }
           const gradle = this.jhipsterConfig.buildTool === GRADLE;
           const command = gradle ? './gradlew' : './npmw';
-          const args = gradle ? ['npm_install'] : ['install', '--no-audit'];
+          const args = gradle ? ['npm_install'] : ['install'];
```

Comment:
Should we switch back to `npmInstall`?
```suggestion
          const args = gradle ? ['npmInstall'] : ['install'];
```

Reply:
You are the expert.